### ISSUE TITLE
WARNING ... core-js@<3 is no longer maintained 

### ISSUE THREAD

**Issue by Neneil94** [diegomura/react-pdf#1013]
I get this warning while resolving packages (using `yarn install`)

`warning @react-pdf/renderer > @react-pdf/textkit > babel-runtime > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.`

In package.json:

```
...
"dependencies": {
    ...
    "@react-pdf/renderer": "~1.6.11",
    ...
}
...
```

Steps to reproduce:
1. Go to console or terminal 
2. `yarn install`

Is it possible to update the package to get rid of the WARNING while install?

**Comment by rahulbhanushali** [comment]
I was getting a similar warning for iconv-lite.  Adding iconv-lite to the dependencies solved it.
Possibly the same thing would work for corejs here.

**Comment by diegomura** [comment]
Fixed already! No reference for core-js<3 in the code anymore. Thanks!

**Comment by tinkatinka** [comment]
I'm getting this message with v2.0.15. In my package.json:
````
"dependencies": {
  "@react-pdf/renderer": "^2.0.15"
  ...
````
`npm ls core-js` says:
````
├─┬ @react-pdf/renderer@2.0.15
│ └─┬ @react-pdf/font@2.0.10
│   └─┬ @react-pdf/fontkit@2.0.6
│     └─┬ dfa@1.1.0
│       └─┬ babel-runtime@6.26.0
│         └── core-js@2.6.12
````
So I think the problem comes from the `dfa` dependency in `@react-pdf/fontkit`. Maybe it can be solved by bumping `dfa` to  v1.1.2, which has zero dependencies but is otherwise unchanged wrt v1.1.0, as far as I can see?
